lib/Doctrine/ORM/Cache/Region/DefaultRegion.php

```diff
@@ -100,30 +102,28 @@ public function get(CacheKey $key)
      */
     public function getMultiple(CollectionCacheEntry $collection)
     {
-        $keysToRetrieve = array();
-
-        foreach ($collection->identifiers as $index => $key) {
-            $keysToRetrieve[$index] = $this->name . '_' . $key->hash;
-        }
-
-        $items = array_filter(
-            array_map([$this->cache, 'fetch'], $keysToRetrieve),
-            function ($retrieved) {
-                return false !== $retrieved;
+        $result = array();
```

Comment:
Yeah, I know and I know Doctrine uses a lot of early returns. Not my favorite though, I prefer the 'one return per function' approach. 

Not sure whether it is a formal Doctrine code convention?